packages/graphql/tsup.config.ts

```diff
@@ -4,4 +4,5 @@ export default defineConfig({
   entry: ['src/index.ts', 'src/ops.ts', 'src/util.ts'],
   clean: true,
   dts: true,
+  publicDir: 'src/public',
```

Comment:
very useful -> allows us to automatically copy files from the `src/public` directory into `dist` on each build or dev generation (replacement for manual copy.js (could replace in `prisma` package as well). the client and server JSON are generated into the public folder and copied from there to dist.